src/Components/Navbar/Navbar.tsx

```diff
@@ -35,14 +55,14 @@ const Navbar = () => {
           <Box className="navAvatar">
             <Tooltip title="Open settings">
               <IconButton>
-                <Avatar alt="Remy Sharp" src="/static/images/avatar/2.jpg" />
+                <Avatar src={userData?.profileImage == null ? defaultImage : `http://upskilling-egypt.com:3000/` + userData?.profileImage} alt="Remy Sharp" />
```

Comment:
We don't need to add a default image. The Avatar component will be set to the image, and if it fails to load it, it will fall back to the first letter if alt attribute is provided, otherwise, it will be set to default avatar image.

https://mui.com/material-ui/react-avatar/#fallbacks
![image](https://github.com/mohamedsaeid1998/Hotel-Management-System-HMS/assets/64327685/b8764b34-6776-4316-a768-6c0d5b44bfaf)

Fallback to first letter in the alt attribute:
![image](https://github.com/mohamedsaeid1998/Hotel-Management-System-HMS/assets/64327685/d03046cd-510f-4cd5-b18c-7aa08a78d541)

Fallback to avatar image:
![image](https://github.com/mohamedsaeid1998/Hotel-Management-System-HMS/assets/64327685/a0bdf342-7983-4e9d-ab08-8c26ff64ba68)